ReactiveSwiftTests/LifetimeSpec.swift

```diff
@@ -1,6 +1,6 @@
 import Quick
 import Nimble
-import ReactiveCocoa
+import ReactiveSwift
 import Result
 
 final class LifetimeSpec: QuickSpec {
```

Comment:
The description perhaps should be changed, and be duplicated for `NSObject.rac_lifetime` in ReactiveCocoa instead?